### DISCUSSION TITLE
SIZES AND STYLES DOESNT WORK

### DISCUSSION THREAD

**Discussion by wesleyso2025** [primefaces/community#2758] [PrimeNG]
In the example the controls are seen as select text boxes among other smaller ones but when implementing them they look very large. How do I fix that?

Additionally, when trying to import the SVG icons for the buttons, an import error appears.

I am sending the images.
![real1](https://github.com/user-attachments/assets/68981d27-2900-4217-954c-af32ebc8b160)
![real2](https://github.com/user-attachments/assets/aad0d07b-cacc-46f6-8339-84980cb5c5b7)

![image](https://github.com/user-attachments/assets/d761f7a9-78b4-4577-a372-ebe345dc56f3)

**Comment by qwadrox** [comment]
Hi @wesleyso2025. If I understood you correctly, your problem is that the fields look larger in your application than in the documentation. This is because the documentation site has a global font-size config set to 14px (default is 16px in browsers): 
```html
<html lang="en" style="font-size: 14px;" 
</html>
```
Here are the docs regarding this: https://primeng.org/theming#scale 

> If you require bigger or smaller components, just change this variable and components will scale accordingly


You forgot to write down the error message about icons what you mentioned, but you can check out the corresponding docs for help: https://primeng.org/icons#size
